api/src/main/java/org/accula/api/handlers/PullsHandler.java

```diff
@@ -43,12 +42,15 @@
                     final var projectId = Long.parseLong(request.pathVariable(PROJECT_ID));
                     final var pullNumber = Integer.parseInt(request.pathVariable(PULL_NUMBER));
 
-                    return pullRepo.findByNumber(projectId, pullNumber)
+                    return pullRepo
+                            .findByNumber(projectId, pullNumber)
                             .switchIfEmpty(Mono.error(PULL_NOT_FOUND_EXCEPTION))
-                            .flatMap(pull -> ServerResponse
-                                    .ok()
-                                    .contentType(APPLICATION_JSON)
-                                    .bodyValue(modelToDtoConverter.convert(pull)));
+                            .flatMap(pull -> Mono.just(pull)
```

Comment:
redundant `pull -> Mono.just(pull)`

This can be simplified to:
```java
...
.switchIfEmpty(Mono.error(PULL_NOT_FOUND_EXCEPTION))
.zipWith(pullRepo.findPrevious(projectId, pullNumber, pull.getAuthor().getId()).collectList())
                                    .flatMap(pullWithPrev -> ServerResponse
                                            .ok()
                                            .contentType(APPLICATION_JSON)
                                            .bodyValue(ModelToDtoConverter.convert(pullWithPrev.getT1(), pullWithPrev.getT2()))));
...
```

Reply:
Точно хочется возится с getT1, getT2?

Reply:
Okay, I missed that